openai_sdk_agent_app_sample_code/tools/test_code_exec_as_uc_tool.py

```diff
@@ -1,10 +1,15 @@
 import pytest
 from cookbook.tools.uc_tool import UCTool
+import os
 
 CATALOG = "ep"
 SCHEMA = "cookbook_local_test"
 
 
+if os.getenv("GITHUB_ACTIONS") == "true":
```

Comment:
As an alternative, we could mock out the WorkspaceClient APIs & DatabricksFunctionClient in these tests, but skipping these in CI for now for simplicity. It also seems important to actually be able to test UC tool execution against a real backend, given that there are currently differences between local & remote execution. If there weren't, I guess we could just test the actual function definition and assume UC execution works fine

Reply:
+1